crates/cxx-qt-lib-headers/include/gui/qfont.h

```diff
@@ -0,0 +1,31 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#pragma once
+
+#include <cinttypes>
```

Comment:
do you need this include? guess std::true_type might need type_traits? and then this might need to be moved to the cpp file ?